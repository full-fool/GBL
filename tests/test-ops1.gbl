class mymain extends Main{
int main()
{
  printlni(1 + 2);
  printlni(1 - 2);
  printlni(1 * 2);
  printlni(100 / 2);
  printlni(99);
  printlnb(1 == 2);
  printlnb(1 == 1);
  printlni(99);
  printlnb(1 != 2);
  printlnb(1 != 1);
  printlni(99);
  printlnb(1 < 2);
  printlnb(2 < 1);
  printlni(99);
  printlnb(1 <= 2);
  printlnb(1 <= 1);
  printlnb(2 <= 1);
  printlni(99);
  printlnb(1 > 2);
  printlnb(2 > 1);
  printlni(99);
  printlnb(1 >= 2);
  printlnb(1 >= 1);
  printlnb(2 >= 1); 
  return 0;
}
}