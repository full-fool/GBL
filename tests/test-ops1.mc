int main()
{
  println(1 + 2);
  println(1 - 2);
  println(1 * 2);
  println(100 / 2);
  println(99);
  println(1 is 2);
  println(1 is 1);
  println(99);
  println(1 neq 2);
  println(1 neq 1);
  println(99);
  println(1 < 2);
  println(2 < 1);
  println(99);
  println(1 <= 2);
  println(1 <= 1);
  println(2 <= 1);
  println(99);
  println(1 > 2);
  println(2 > 1);
  println(99);
  println(1 >= 2);
  println(1 >= 1);
  println(2 >= 1); 
  return 0;
}
