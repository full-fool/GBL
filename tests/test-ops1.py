import sys
class mymain:
    def __init__(self):
        pass

    def main(self):
        print ((1) + (2))
        print ((1) - (2))
        print ((1) * (2))
        print ((100) / (2))
        print (99)
        print ((1) == (2))
        print ((1) == (1))
        print (99)
        print ((1) != (2))
        print ((1) != (1))
        print (99)
        print ((1) < (2))
        print ((2) < (1))
        print (99)
        print ((1) <= (2))
        print ((1) <= (1))
        print ((2) <= (1))
        print (99)
        print ((1) > (2))
        print ((2) > (1))
        print (99)
        print ((1) >= (2))
        print ((1) >= (1))
        print ((2) >= (1))
        return 0
        pass


if __name__ == "__main__":
    _main = mymain()
    _main.main()

