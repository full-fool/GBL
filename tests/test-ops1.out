3
-1
2
50
99
False
True
99
True
False
99
True
False
99
True
True
False
99
False
True
99
False
True
True
